manager/rbac_manager.py

```diff
@@ -96,7 +99,38 @@ def _parse_permissions(self, response: dict) -> [RbacPermission]:
             res.append(self._parse_permission(perm_raw["permission"]))
         return res
 
-    def fetch_permissions(self, x_rh_identity: str) -> [RbacPermission]:
+    @staticmethod
+    def _find_host_groups(role_list) -> List[str]:
+        """Retreive list of inventory groups from RBAC response"""
+        res = []
+        for role in role_list:
+            if 'permission' not in role:
```

Comment:
do we really need to parse the response, can't we assume that the schema is correct and it always has `permission: string` and `resourceDefinitions: list` fields?

then it can be simplified as something like
```python
for role in role_list:
    if role.get("permission") != "inventory:hosts:read":
        continue

    for rscdef in role.get("resourceDefinitions", []):
        attrfilter = rscdef.get("attributeFilter", {})
        if attrfilter.get("key") != "group.id":
            continue
        res.extend(attrfilter.get("value", []))
```